src/main/java/org/seqdoop/hadoop_bam/BAMSplitGuesser.java

```diff
@@ -111,7 +111,8 @@ public long guessNextBAMRecordStart(long beg, long end)
 		// the headers may exceed MAX_BYTES_READ in length
 		if (beg == 0) {
 			this.inFile.seek(beg);
-			SamReader open = SamReaderFactory.makeDefault().open(SamInputResource.of(inFile));
+			SamReader open = SamReaderFactory.makeDefault().setUseAsyncIo(false)
```

Comment:
@tomwhite I think it would be good to have a comment here that this method does not close the stream, which means it can be used by subsequent calls to the same method. This will help future readers of this code